### PR TITLE
[2017-02][process] Disable tests that do not work on mobile

### PR DESCRIPTION
**Pull request by luhenry**
This fixes bug https://bugzilla.xamarin.com/show_bug.cgi?id=53881

### CODE REVIEW

File: mcs/class/System/Test/System.Diagnostics/ProcessTest.cs [hunk 1]
```diff
@@ -1116,13 +1116,15 @@ public void TestExitedRaisedTooSoon ()
 #endif // MONO_FEATURE_PROCESS_START
 
 		[Test]
+		[NUnit.Framework.Category ("MobileNotWorking")]
```
Comment (marek-safar) [inline mcs/class/System/Test/System.Diagnostics/ProcessTest.cs]: What is failing and on what mobile feature is causing this?

Comment (luhenry) [inline mcs/class/System/Test/System.Diagnostics/ProcessTest.cs]: This fixes bug https://bugzilla.xamarin.com/show_bug.cgi?id=53881

Comment (marek-safar) [inline mcs/class/System/Test/System.Diagnostics/ProcessTest.cs]: This looks like to be `#if MONOTOUCH` specific.

File: mcs/class/System/Test/System.Diagnostics/ProcessTest.cs [hunk 2]
```diff
@@ -1146,6 +1148,7 @@ public void NonChildProcessWaitForExit ()
 		}
 
 		[Test]
+		[NUnit.Framework.Category ("MobileNotWorking")]
```
Comment (marek-safar) [inline mcs/class/System/Test/System.Diagnostics/ProcessTest.cs]: Only this one fails with pretty odd message ` System.InvalidOperationException : Process has exited, so the requested information is not available.` Are you sure that's intended outcome?

**Comment by kumpera** [comment]
WTF @marek-safar!!! I have a PR with that sitting in your inbox for 2+ weeks and you didn't bother approving it on petty changes. 😢

**Pull request merged**